tests/files/tests.py

```diff
@@ -169,6 +169,21 @@ def test_io_wrapper(self):
             test_file.seek(0)
             self.assertEqual(test_file.read(), (content * 2).encode())
 
+    def test_file_lock(self):
+        file_path = Path(__file__).parent / 'test.png'
+        with open(file_path) as f:
+            self.assertIs(locks.lock(f, locks.LOCK_EX), True)
+
+        self.assertIs(locks.lock(22, locks.LOCK_EX), False)
```

Comment:
I would rather try to create a second non-blocking lock:

```suggestion
        file_path = Path(__file__).parent / 'test.png'
        f1 = open(file_path)
        f2 = open(file_path)
        self.assertIs(locks.lock(f1, locks.LOCK_EX), True)
        self.assertIs(locks.lock(f2, locks.LOCK_EX | LOCK_NB), False)
```